vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_uefi.rs

```diff
@@ -231,19 +235,22 @@ async fn nvme_relay_heuristic_16vp_768mb_heavy(
 /// There _should_ be enough private pool memory for the NVMe driver to
 /// allocate all of its buffers contiguously.
 #[openvmm_test(openhcl_uefi_x64[nvme](vhd(ubuntu_2504_server_x64)))]
-async fn nvme_relay_32vp_500mb_very_heavy(
+async fn nvme_relay_32vp_768mb_very_heavy(
     config: PetriVmBuilder<OpenVmmPetriBackend>,
 ) -> Result<(), anyhow::Error> {
     nvme_relay_test_core(
-        config,
+        config.with_memory(MemoryConfig {
+            startup_bytes: 16 * 1024 * 1024 * 1024,
```

Comment:
If I'm reading the source correctly, the default is 4GB and you're upping it to 16GB here.
1. Out of curiosity, what failed with the defaults?
2. Will all tests machines permit 16GB VMs?

Reply:
You are correct in how you read this. I had thought that the 32VP / 4GB RAM test was causing instability, but I actually think the root cause is #2347 . That being said, we don't have anyone that actually wants to use 32 VPs with just 4GB of RAM.

For #2 - yes, all test machines will support 16GB VMs. I'm pretty confident. But our pools are homegenous, so if CI passes then empirically yes :). Great questions.